javatests/arcs/core/data/proto/AccessPathProtoDecoderTest.kt

```diff
@@ -0,0 +1,53 @@
+package arcs.core.data.proto
+
+import arcs.core.data.AccessPath
+import arcs.core.data.HandleConnectionSpec
+import arcs.core.data.HandleMode
+import arcs.core.data.TypeVariable
+import arcs.core.testutil.assertThrows
+import arcs.core.testutil.fail
+import com.google.common.truth.Truth.assertThat
+import com.google.protobuf.TextFormat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Parses a given proto text as [AccessPathProto]. */
+fun parseAccessPathProto(protoText: String): AccessPathProto {
+    val builder = AccessPathProto.newBuilder()
+    TextFormat.getParser().merge(protoText, builder)
+    return builder.build()
+}
+
+@RunWith(JUnit4::class)
+class AccessPathProtoDecoderTest {
+    @Test
+    fun decodesAccessPathNoSelectors() {
+        val protoText = """
+        handle_connection: "input"
```

Comment:
Minor nit: I would indent this one level more so it's offset from the other code.

Reply:
Done.